src/wallet/coin_selection.rs

```diff
@@ -114,8 +114,9 @@ pub type DefaultCoinSelectionAlgorithm = BranchAndBoundCoinSelection;
 #[cfg(test)]
 pub type DefaultCoinSelectionAlgorithm = LargestFirstCoinSelection; // make the tests more predictable
 
-// Base weight of a Txin, not counting the weight needed for satisfaying it.
-// prev_txid (32 bytes) + prev_vout (4 bytes) + sequence (4 bytes) + script_len (1 bytes)
+/// Base weight of a Txin, not counting the weight needed for satisfaying it
```

Comment:
Looks like you don't have spell check enabled in your editor, you should be able to configure your editor to spell check just the comments.

Reply:
Thanks for the catch, I have spell checking on but I guess I missed the warning on that one. Fixed now.